packages/react/src/components/Tooltip/next/__tests__/Tooltip-test.js

```diff
@@ -55,3 +57,42 @@ describe('Tooltip', () => {
     expect(screen.getByText('test')).toHaveAttribute('aria-describedby');
   });
 });
+
+describe('DefintiionTooltip', () => {
```

Comment:
I think we should split this out into its own test file at `next/__tests__/DefinitionTooltip-test.js` to make it easier to find 👍

Reply:
On it